cwbi-auth-http-client/src/main/java/hec/army/usace/hec/cwbi/auth/http/client/trustmanagers/CwbiAuthTrustManager.java

```diff
@@ -45,8 +46,7 @@ private static X509TrustManager buildTrustManager() {
             trustManagerFactory.init(ts);
             retVal =  new CwbiAuthTrustManager(trustManagerFactory);
         } catch (CertificateException | NoSuchAlgorithmException | KeyStoreException | IOException e) {
-            String errorMsg = "Error retrieving X509TrustManager: " + e.getMessage();
-            LOGGER.log(Level.SEVERE, errorMsg, e);
+            LOGGER.log(Level.SEVERE, "Error retrieving X509TrustManager", e);
```

Comment:
This will be logged as severe and go to a user if there are problems. What does the error message mean to a user? I.e. take Shannon or Kevin as a user actor and give them an example of them getting this error and work on the message until they understand it.

Reply:
Updated to "Unable to authenticate with CWBI Auth server". Ran that by Kevin as well and it made sense to him.